packages/ckeditor5-table/src/tableclipboard.ts

```diff
@@ -62,7 +62,7 @@ export default class TableClipboard extends Plugin {
 	 * @inheritDoc
 	 */
 	public static get requires() {
-		return [ TableSelection, TableUtils ] as const;
+		return [ 'ClipboardPipeline', TableSelection, TableUtils ] as const;
```

Comment:
This plugin is available in the core DLL so you can use constructor reference like here: 

https://github.com/ckeditor/ckeditor5/blob/4c08d35c2e7dc5fef4fe2cbc97c5db7e72d3a7a2/packages/ckeditor5-image/src/image/imageblockediting.ts#L45-L47

https://github.com/ckeditor/ckeditor5/blob/4c08d35c2e7dc5fef4fe2cbc97c5db7e72d3a7a2/packages/ckeditor5-image/src/image/imageblockediting.ts#L11

Reply:
Good point, thx!